src/client.rs

```diff
@@ -636,64 +637,55 @@ pub extern "C" fn rustls_client_session_read(
     }
 }
 
-/// Read up to `count` TLS bytes from `buf` (usually read from a socket) into
-/// the ClientSession. This may make packets available to
-/// `rustls_client_session_process_new_packets`, which in turn may make more
-/// bytes available to `rustls_client_session_read`.
-/// On success, store the number of bytes actually read in *out_n (this may
-/// be less than `count`). This function returns success and stores 0 in
-/// *out_n when the input count is 0.
+/// Read some TLS bytes from the network into internal buffers. The actual network
```

Comment:
nit: I think you could avoid explicitly mentioning the network because you're not necessarily reading or writing to/from a network.

Reply:
I think "network" is nicely evocative here, and folks who are reading/writing from other buffers instead will get the gist.